aria/orchestrator/context/common.py

```diff
@@ -38,10 +38,27 @@ class BaseContext(object):
     """
 
     INSTRUMENTATION_FIELDS = (
+        modeling.models.Service.inputs,
```

Comment:
Because you should be able to do things like: {{ctx node capabilities host properties username}}

The ctx proxy should let you access all models.